cli/e2e-tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+kind create cluster
+go test -v ./tests/ --timeout 30m
+kind delete cluster
```

Comment:
Maybe we should put this in a trap so it happens if the test is cancelled  or interrupted with ctrl-c or whatnot?